bungee/luckperms/build.gradle

```diff
@@ -16,11 +16,9 @@
  */
 
 repositories {
-    maven {
-        url "https://repo.lucko.me/"
-    }
+    mavenCentral()
```

Comment:
No need for this one, maven central is configured in the root build.gradle https://github.com/CodeCrafter47/minecraft-data-api/blob/master/build.gradle#L42

Reply:
Fixed